rpms/must-gather-clean.spec

```diff
@@ -0,0 +1,100 @@
+export GOPATH=%{gopath}
+cd %{gopath}/src/github.com/openshift/must-gather-clean
+go mod edit -go=%{golang_version}
+%ifarch x86_64
+# go test -race is not supported on all arches
+GOFLAGS='-mod=vendor' make test
+%endif
+make prepare-release
+echo "%{mgc_version}" > dist/release/VERSION
+unlink %{gopath}/src/github.com/openshift/must-gather-clean
+
+%install
+mkdir -p %{buildroot}/%{_bindir}
+install -m 0755 dist/bin/linux-`go env GOARCH`/must-gather-clean %{buildroot}%{_bindir}/must-gather-clean
+mkdir -p %{buildroot}%{_datadir}
+install -d %{buildroot}%{_datadir}/%{name}-redistributable
+install -p -m 755 dist/release/must-gather-clean-linux-amd64 %{buildroot}%{_datadir}/%{name}-redistributable/must-gather-clean-linux-amd64
+install -p -m 755 dist/release/must-gather-clean-linux-arm64 %{buildroot}%{_datadir}/%{name}-redistributable/must-gather-clean-linux-arm64
+install -p -m 755 dist/release/must-gather-clean-linux-ppc64le %{buildroot}%{_datadir}/%{name}-redistributable/must-gather-clean-linux-ppc64le
+install -p -m 755 dist/release/must-gather-clean-linux-s390x %{buildroot}%{_datadir}/%{name}-redistributable/must-gather-clean-linux-s390x
+install -p -m 755 dist/release/must-gather-clean-darwin-amd64 %{buildroot}%{_datadir}/%{name}-redistributable/must-gather-clean-darwin-amd64
+install -p -m 755 dist/release/must-gather-clean-windows-amd64.exe %{buildroot}%{_datadir}/%{name}-redistributable/must-gather-clean-windows-amd64.exe
+cp -avrf dist/release/must-gather-clean*.tar.gz %{buildroot}%{_datadir}/%{name}-redistributable
+cp -avrf dist/release/must-gather-clean*.zip %{buildroot}%{_datadir}/%{name}-redistributable
+cp -avrf dist/release/SHA256_SUM %{buildroot}%{_datadir}/%{name}-redistributable
+cp -avrf dist/release/VERSION %{buildroot}%{_datadir}/%{name}-redistributable
+
+%files
+%license LICENSE
+%{_bindir}/must-gather-clean
+
+%package redistributable
+Summary:        %{product_name} client CLI binaries for Linux, macOS and Windows
+BuildRequires:  gcc
+BuildRequires:  golang >= %{golang_version}
+Provides:       %{package_name}-redistributable = %{mgc_version}
+Obsoletes:      %{package_name}-redistributable <= %{mgc_version}
+
+%description redistributable
+%{product_name} client must-gather-clean cross platform binaries for Linux, macOS and Windows.
+
+%files redistributable
+%license LICENSE
+%dir %{_datadir}/%{name}-redistributable
+%{_datadir}/%{name}-redistributable/must-gather-clean-linux-amd64
+%{_datadir}/%{name}-redistributable/must-gather-clean-linux-amd64.tar.gz
+%{_datadir}/%{name}-redistributable/must-gather-clean-linux-arm64
+%{_datadir}/%{name}-redistributable/must-gather-clean-linux-arm64.tar.gz
+%{_datadir}/%{name}-redistributable/must-gather-clean-linux-ppc64le
+%{_datadir}/%{name}-redistributable/must-gather-clean-linux-ppc64le.tar.gz
+%{_datadir}/%{name}-redistributable/must-gather-clean-linux-s390x
+%{_datadir}/%{name}-redistributable/must-gather-clean-linux-s390x.tar.gz
```

Comment:
could you also remove the ppc and s3 builds please? just talked with the odo manager and they mentioned that was just a specific IBM release that they also needed to specifically support and they got assistance when building against it.

Reply:
Removed.